### DISCUSSION TITLE
PBChannelAddUser: confuso los campos de la rutina

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#382] [2. Questions and Answers] [answered]
Hola!

Estaba usando el pbBot para invitar un usuario al canal por property, y encontré confusa la descripción de los campos. 
![Screen Shot 2021-09-01 at 09 55 37](https://user-images.githubusercontent.com/42095617/131684521-1144985a-6eeb-4957-936a-ac16d042020f.png)
![Screen Shot 2021-09-01 at 09 56 03](https://user-images.githubusercontent.com/42095617/131684542-f810ea65-0362-4e21-a89b-ebc3203ac06c.png)

Como quiero agregar un usuario por propiedad, no sé cuál será su id y no debería ser necesario que tenga un accessRole específico. Entonces, como aprece en la descripción, solo rellenaría la opción de property y no debería rellenar los campos de access rol ni  user.
Sin embargo, dichos campos aparecen como requeridos. Entonces rellene con vacío el campo de accessrole y el id del user  que respondió el formulario (porque si lo dejaba vacío, me aparecía error). 

Por tanto, encuentro que es confuso lo que hay que hacer para que funcione.

¿Qué piensan? 

Contexto: Cotalker Local

**Comment by NicolasPLG** [comment, marked as answer]
@ValeMP prueba con colocar null en los campos que no necesitas 0>. en el addtaskeditor version 1.0.0, se colocaba null para no considerar el tipo respectivo. Si el contexto es de una task, te recomiendo usar el taskaddeditor (ahora los elementos no son obligatorios), porque si usas el invite user to channel de un channel tasks, en el momento que exsten algun cambio en la task (cambiar de estado, nombre, etc), el usuario que se agregó se saldrá.

**Reply by ValeMP** [reply]
gracias nico!

**Reply by ValeMP** [reply]
cambiando el bot al editar usuarios de la task, ya no se presentaba el mismo problema!
